docs/APIReference-AtomicBlockUtils.md

```diff
@@ -24,3 +24,21 @@ insertAtomicBlock: function(
   character: string
 ): EditorState
 ```
+
+### moveAtomicBlockBefore
+
+```
+moveAtomicBlockBefore: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
+
+### moveAtomicBlockAfter
+
+```
+moveAtomicBlockAfter: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
```

Comment:
These functions are a bit confusing. Am I moving the specified `ContentBlock` or am I moving something before/after the specified `ContentBlock`?

I think the best thing for API clarity is to specify which block is being moved and which block is the target to move it before/after.